apps/web/src/pages/Home/IAPHomePage/Home.tsx

```diff
@@ -97,29 +82,57 @@ const Home = () => {
           alt=""
         />
         <div
-          className="hero-logo"
+          data-h2-transform="p-tablet(translate(-50%, 0))"
           data-h2-background-color="base(primary.darker) p-tablet(transparent)"
           data-h2-padding="base(x1.2, x2, x1, x2) p-tablet(0)"
           data-h2-layer="base(1, relative) p-tablet(1, absolute)"
           data-h2-order="base(1) p-tablet(2)"
           data-h2-location="p-tablet(5%, auto, auto, 50%)"
-          data-h2-width="base(100%) p-tablet(40vw)"
+          data-h2-width="base(100%) p-tablet(40em) l-tablet(55em)"
+          data-h2-display="base(flex)"
+          data-h2-flex-direction="base(column) p-tablet(row)"
+          data-h2-align-items="base(center) p-tablet(flex-start)"
+          data-h2-justify-content="base(center)"
```

Comment:
I think adding ` data-h2-align-items="base(center)"` would make this look even better, but that's just me.